packages/ndla-ui/src/MessageBox/MessageBox.tsx

```diff
@@ -111,28 +136,36 @@ const CloseButtonText = styled.span`
     font-weight: ${fonts.weight.normal};
   }
 `;
+const LinkWrapper = styled.div`
+  display: block;
+  width: 100%;
+  background-color: #f9f4c8;
+  padding-bottom: 20px;
+  padding-left: 56px;
+`;
+const Link = styled.a`
+  color: blue;
+  font-size: 16px;
+  padding-bottom: 5px;
+  margin: 0px 50px 1px 5px;
+`;
 
 type Props = {
   type?: WrapperProps['boxType'];
   heading?: string;
   sticky?: boolean;
   onClose?: () => void;
-  children: React.ReactNode;
+  children?: React.ReactNode;
+  links?: []; //Takes and object with a name and a link. Eks: {'link1', 'www.link.no'}
```

Comment:
Definer heller links-objektet ordentlig. Dette gir ingen typesikkerhet. Finnes det en type allerede som kan brukes?
Dessuten om du endrer første linje til `import React, { ReactNode } from 'react';` kan du fjerne React fra children.
Fila mangler også lisens så det er fint om du legger til. Sjekk historikken på github for årstall for fila.